tooling/templatize/pkg/config/types.go

```diff
@@ -11,6 +12,24 @@ type configProviderImpl struct {
 
 type Variables map[string]any
 
+func (v Variables) GetByPath(path string) (any, bool) {
```

Comment:
This was precisely the thing I wished we would not need to write. `{{ .parent["child.foo"].grandchild }}` will break something like this. The quicker we stop supporting an `env` list, the better.

Reply:
+1